src/pretalx/submission/forms/question.py

```diff
@@ -31,8 +33,11 @@ def __init__(self, *args, **kwargs):
         self.queryset = Question.all_objects.filter(event=self.event, active=True)
         if self.target_type:
             self.queryset = self.queryset.filter(target=self.target_type)
+            if self.submission_type:
```

Comment:
This should only happen if the `target_type` is submission, to be sure we do the correct thing here.